environment/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[project]
+name = "aind-vr-foraging-primary-data-nwb-packaging"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.12"
+dependencies = [
+    "aind-behavior-vr-foraging[data]",
+    "aind-data-schema>=1.4.0",
```

Comment:
This is locked via aind-behavior-vr-foraging. You can use the `[dev]` as a way to get all these requirements.